python/private/py_wheel.bzl

```diff
@@ -275,6 +278,8 @@ def _py_wheel_impl(ctx):
         metadata_contents.append("Home-page: %s" % ctx.attr.homepage)
     if ctx.attr.license:
         metadata_contents.append("License: %s" % ctx.attr.license)
+    if ctx.attr.description_content_type:
```

Comment:
How about setting the content type based on the file extension if it isn't specified? It looks like there's just .md, .rst, and .txt that need to be checked for.

Reply:
Done. PTAL